src/Core/ConfigurationDefaults.re

```diff
@@ -36,6 +36,12 @@ let getDefaultConfigString = configName =>
   "workbench.sideBar.location": "left",
   "workbench.sideBar.visible": true,
   "workbench.statusBar.visible": true,
+  "workbench.statusBar.items": {
+    "left": ["notificationCount", "macro", "leftItems", "diagnosticCount", "git"],
```

Comment:
Instead of `left`/`right`, we'll need to change these to `start`/`end` to be in sync with documentation